superset-frontend/src/components/Select/Select.tsx

```diff
@@ -299,13 +300,13 @@ const Select = (
     onError,
     onChange,
     onClear,
-    onDropdownVisibleChange,
```

Comment:
This prop should stay or remove the code that uses it.

Reply:
This property is used and should be restored.

Reply:
@michael-s-molina 
Resolved